protos/holding_transaction.proto

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+import "organization.proto";
+
+message CreateHoldingAction {
+
+	// Unique identifier of the organization which owns this holding
+	string owner_id = 1;
```

Comment:
I wonder if we need this field? Because this should be the id for the organization of the participant who submitted the transaction. I don't know if it makes a huge different either way.

Reply:
True, the information of the organization which has the asset would be accessible through the participant that signed the transaction. Seems a bit redundant